impl/src/main/resources/META-INF/permittedTaglibs-1_1.tld

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+
+    Copyright (c) 2010, 2020 Oracle and/or its affiliates. All rights reserved.
+    Copyright (c) 2022-2022 Contributors to the Eclipse Foundation
+
```

Comment:
Was this copyright added because we made updates here but have since removed them and they are in the new permittedTagLibs.tld?

Reply:
Yes, made changes to this file but later undid them. I guess we don't need the copyright anymore.

Reply:
I copied the original to ensure it is the same as before.